source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_driver.jsonnet

```diff
@@ -0,0 +1,45 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+{
+  type: 'Question',
+  id: 'anyone-else-driver',
+  question: {
+    id: 'anyone-else-driver-question',
+    title: {
+      text: 'I will now collect the names of everyone who lived at that address on Sunday {census_date}',
+      placeholders: [
+        placeholders.censusDate,
+      ],
+    },
+    type: 'General',
+    answers: [
+      {
+        id: 'anyone-else-driver-answer',
+        mandatory: false,
```

Comment:
This should be made mandatory, as we cannot route to the add screen when they haven't answered. Already checked with Donna.

Reply:
Done, thanks!👍